src/common/Error/styled.js

```diff
@@ -0,0 +1,57 @@
+import styled from "styled-components";
+import { NavLink } from "react-router-dom";
+
+export const Wrapper = styled.div`
+    margin: 0 auto;
+    max-width: 630px;
+    text-align: center;
+    padding: 16px;
+`;
+
+export const DangerImage = styled.img`
+    @media(max-width: ${({theme}) => theme.breakpoints.xsmall}px) {
+        width: 100px;
+    }
+`
+
+export const Title = styled.h1`
+    margin: 24px 0;
+    font-size: 36px;
+    line-height: 120%;
+
+    @media(max-width: ${({theme}) => theme.breakpoints.xsmall}px) {
+        font-size: 20px;
+        margin: 16px 0;
+    }
```

Comment:
If you make changes at the beginning of the code,  three lines of code will be enough here. Without @media.

font-weight: 600; font-size: 36px; line-height: 1.2;